posts/dask-summit-takeaway.md

```diff
@@ -0,0 +1,120 @@
+- STAC (spatio-temporal asset catalog) can be helpful when working with satellite imagery, making it easy to search for specific datasets
+  - Similar to intake, able to search spatially as well
+
+### [Pangeo Session Talk Links](https://summit.dask.org/schedule/presentation/1/pangeo/)
+
+- Chelle: notebook can be found in folder AWS-notebooks
+  - [Jupyter Notebooks](https://github.com/pangeo-gallery/osm2020tutorial)
+- Julius:
+  - [Slides](https://speakerdeck.com/jbusecke/dask-and-the-ocean-death-zones-lessons-from-a-real-life-earth-science-workflow-with-a-fullish-pangeo-stack)
+  - [CMIP6 Prepocessing](https://github.com/jbusecke/cmip6_preprocessing)
+  - [Oxygen Minimum Zones Paper Preprint](https://www.essoar.org/doi/10.1002/essoar.10507050.1)
+- Rob
+  - [Info about STAC specification](https://stacspec.org)
+- Deepak:
+  - [Dask Groupby Repository](https://github.com/dcherian/dask_groupby)
+- Haiying:
+  - [Pangeo Benchmarking Repository](https://github.com/pangeo-data/benchmarking)
+- Damien:
+  - [Software Carpentries Lessons](https://github.com/carpentrieslab/python-aos-lesson)
+
+## [Growth and History of Anaconda + Numpy + SciPy + Dask](https://zoom.us/rec/share/_KQiagnZBnW0xPIHAHZZs553JuFsJXfkQpdwUmTca7-QE91safAQCefSYr6-Kz3R.EDHHYf_864Dw3v8Q?startTime=1621515755000)
+
+This session was a keynote focused on how these packages + ecosystem grew
+
+- Key question - how do you get from a few passionate few to productive many?
+  - Keep your motivated few to less than 5 people
+- Where efforts should be focused
+  - Dedicating full time employees to work on community dev projects
+  - Build the community as you go - this is essential
+    - Sometimes it takes a while to build up your product... that's okay!
+- Working code solves most arguements
+  - At the end of the day, someone has to sit down and code something up.. if it works, great!
+- Great projects start with a need
+  - The conceptual idea is not always important - specific use cases are more important
+- You need to solve **_something_**
+  - This may require shrinking the scope to a more manageable size
+- Let teams go and do their own thing
+  - Taken scope -> give to others -> do that thing
+- You need to be humble with your project, but not too humble; simple but not too simple
+  - It is all about balance
+- Top down approaches typically do not work
+  - You will always want more resources...
+- The power of Python is the community
+  - Key idea - **_there are more smart people outside of your organization than inside your organization_**
+  - If you try to do everything, it will inevitably fail - need to make it more simple
+  - Unlock the community - entrain users, become contributors
+- **_Provide scope and build people_**
+  - Move towards "grassroots" not so much top down
+
+## Xarray User Forum + Updates
+
+- Xarray now includes a backend api, allowing contributors to add new backends for reading + operating on datasets
```

Comment:
domain-specific dataset reading code and live outside xarray but be used through `open_dataset`.